backend/apps/mentorship/management/commands/sync_mentorship_issue_comments.py

```diff
@@ -0,0 +1,124 @@
+"""Syncs comments for issues relevant to active mentorship programs."""
+
+import logging
+import re
+
+from django.core.management.base import BaseCommand
+from github.GithubException import GithubException
+
+from apps.github.auth import get_github_client
+from apps.github.common import sync_issue_comments
+from apps.github.models.issue import Issue
+from apps.mentorship.models import ParticipantInterest, Program
+
+logger = logging.getLogger(__name__)
+
+INTEREST_PATTERNS = [
```

Comment:
Let's not overcomplicate it and use `/interested` as the only pattern/command for now.